src/org/opentosca/containerapi/client/impl/OpenTOSCAContainerInternalAPIClient.java

```diff
@@ -115,11 +99,10 @@ JSONObject getApplicationProperties(final String csarName) {
 	}
 
 	String getAuthor(JSONObject appProps) {
-		if (appProps.has(Constants.OPENTOSCACONTAINERAPI_APPLICATIONRESOURCE_JSON_AUTHORS)) {

```

Comment:
Saw a couple of such changes. I don't see why they are necessary... Please apply our current coding style, whether you like it or not ;-)